sklearn/datasets/tests/test_samples_generator.py

```diff
@@ -148,7 +149,7 @@ def test_make_multilabel_classification_return_indicator():
     for allow_unlabeled, min_length in zip((True, False), (0, 1)):
         X, Y = make_multilabel_classification(n_samples=25, n_features=20,
                                               n_classes=3, random_state=0,
-                                              return_indicator=True,
+                                              return_indicator='dense',
```

Comment:
@jnothman `return_indicator=True` is being called and tested on line 170... do I need an explicit test for it?